README.md

```diff
@@ -1,243 +1,50 @@
moose](https://github.com/zamoose), [@leewillis77](https://github.com/leewillis77), [@imichaeli](https://github.com/imichaeli), [@andrezrv](https://github.com/andrezrv), [@cadwell](https://github.com/cadwell), [@cfoellmann](https://github.com/cfoellmann), [@westi](https://github.com/westi), [@ryanduff](https://github.com/ryanduff), [@selinerdominik](https://github.com/selinerdominik), [@ericandrewlewis](https://github.com/ericandrewlewis), [@vDevices](https://github.com/vDevices), [@sunnyratilal](https://github.com/sunnyratilal), [@enejb](https://github.com/enejb), [@salcode](https://github.com/salcode), [@mattbanks](https://github.com/mattbanks), [@aaroncampbell](https://github.com/aaroncampbell), [@tnorthcutt](https://github.com/tnorthcutt), [@neilpie](https://github.com/neilpie), [@francescolaffi](https://github.com/francescolaffi), [@itsananderson](https://github.com/itsananderson), [@foolswis](https://github.com/foolswis), [@lloydde](https://github.com/lloydde), [@jmbarlow](https://github.com/jmbarlow), [@nacin](https://github.com/nacin), [@thewebists](https://github.com/thewebists), [@iparr](https://github.com/iparr), [@chrishepner](https://github.com/chrishepner), [@miya0001](https://github.com/miya0001), [@iamntz](https://github.com/iamntz), [@mirmillo](https://github.com/mirmillo), [@garyjones](https://github.com/garyjones), [@teraphy](https://github.com/teraphy), [@DrewAPicture](https://github.com/DrewAPicture), [@jjeaton](https://github.com/jjeaton), [@ntwb](https://github.com/ntwb), [@bradp](https://github.com/bradp), [@jb510](https://github.com/jb510), [@pbearne](https://github.com/pbearne), [@robneu](https://github.com/robneu), [@hubdotcom](https://github.com/hubdotcom), [@michaelbeil](https://github.com/michaelbeil), [@EHLOVader](https://github.com/EHLOVader), [@HongPong](https://github.com/HongPong), [@twisty7867](https://github.com/twisty7867), [@khromov](https://github.com/khromov), [@amchoukir](https://github.com/amchoukir), [@grantnorwood](https://github.com/grantnorwood), [@MatthewEppelsheimer](https://github.com/MatthewEppelsheimer), [@cristovaov](https://github.com/cristovaov), [@coreymckrill](https://github.com/coreymckrill), [@jonathanbardo](https://github.com/jonathanbardo), [@rasmusbe](https://github.com/rasmusbe), [@eriktrom](https://github.com/eriktrom), [@f4bsch](https://github.com/f4bsch), [@meitar](https://github.com/meitar), [@keegoid](https://github.com/keegoid), [@LoreleiAurora](https://github.com/LoreleiAurora), [@cobyan](https://github.com/cobyan), [@goblindegook](https://github.com/goblindegook), [@dlackty](https://github.com/dlackty), [@rene-hermenau](https://github.com/rene-hermenau), [@jdevalk](https://github.com/jdevalk), [@kadimi](https://github.com/kadimi), [@sc0ttkclark](https://github.com/sc0ttkclark), [@jonathanbossenger](https://github.com/jonathanbossenger), [@hello-jason](https://github.com/hello-jason), [@weskoop](https://github.com/weskoop), [@thebrandonallen](https://github.com/thebrandonallen)
```

Comment:
Good point, @grappler. That's probably the right answer. I've stuck with this format for so long because I kind of had fun watching the list grow. But we've had a lot of contributors now, so it may be time to change. It would definitely shorten the README. :)